control_panel_api/aws.py

```diff
@@ -46,6 +46,20 @@ def put_bucket_logging(self, name, target_bucket, target_prefix):
                 }
             })
 
+    def put_bucket_tagging(self, name, tags):
+        """Put bucket tagging
+
+        :param name: Bucket name
+        :param tags: List of tags (key, value) e.g. [('buckettype', 'datawarehouse')]
+        :type name: str
+        :type tags: list(tuple)
```

Comment:
(from my little understanding of) S3 tags are key/value pairs. 
So after calling `put_bucket_tagging` you may have something like this attached to the bucket:

```python
{
  'buckettype': 'datawarehouse',
  'other-key': 'other-value',
}
```

I think the `tags` param here should be a dictionary, e.g. you call this like this:

```python
aws.put_bucket_tagging('bucket-name', {'k1': 'v1', 'k2': 'v2'})
```

In case like the above use tuples can lead to confusion, e.g. if I see something like this `('k1', 'v1', 'k2', 'v2')` I may not catch the relationship between `k1`/`v1` and `k2`/`v2`.

(Obviously this means the `for` loop needs to be tweaked but it's a little price to pay for clarity IMHO)

Reply:
Good point, not sure why I went down that route.